tests/src/JIT/Directed/Misc/gettype/gettypetypeofmatrix.csproj

```diff
@@ -22,8 +22,7 @@
     </CodeAnalysisDependentAssemblyPaths>
   </ItemGroup>
   <PropertyGroup>
-    <!-- Set to 'Full' if the Debug? column is marked in the spreadsheet. Leave blank otherwise. -->
-    <DebugType></DebugType>
+    <DebugType>None</DebugType>
```

Comment:
Mono may report 'Windows` for the `RuntimeInformation.IsOSPlatfrom`. If the code still uses that method of platform detection and runs on Mono, it  could cause this.